### ISSUE TITLE
Windows 8.1 x64

### ISSUE THREAD

**Issue by 2010phenix** [valiant1x/intensecoin#16]
Helping to my friend with Intense Wallet and He have problem

upload from GitHub wallet but not work on Windows 8.1 x64 Rus

http://prntscr.com/hy4ule

am try on my computer with Windows7 x64 and it's work correct.

Any solution for Win 8.1?

**Comment by valiant1x** [comment]
please try downloading and installing both Visual C++ VS2017 runtimes:

https://aka.ms/vs/15/release/vc_redist.x86.exe
https://aka.ms/vs/15/release/vc_redist.x64.exe

**Comment by 2010phenix** [comment]
ok thx, tell to him...

**Comment by 2010phenix** [comment]
thay try updae DLL again and not help.... error like write in first post

**Comment by valiant1x** [comment]
please provide an English translation of the error

**Comment by 2010phenix** [comment]
nothing that tell what problem is... but...
----> error when starting the application (address in HEX) to exit the application, click "OK"

**Comment by valiant1x** [comment]
the proper place for wallet GUI bugs is https://github.com/valiant1x/intensecoinwallet/issues

this is likely related to https://github.com/valiant1x/intensecoinwallet/issues/22

**Comment by 2010phenix** [comment]
yes am read about this problem on bitcointalk and ask him... Thay say only english latter in account name and folder wallet
